invokeai/frontend/web/src/services/api/endpoints/images.ts

```diff
@@ -595,10 +698,105 @@ export const imagesApi = api.injectEndpoints({
         }
       },
     }),
+    /**
+     * Upload a multiple images.
+     */
+    uploadMultipleImages: build.mutation<
+      ImageDTO[],
```

Comment:
Mentioned on discord - this query only returns a message now, so the cache handling in `onQueryStarted` is unnecessary (and it errors in js console)